<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialwindow</class>
 <widget class="QWidget" name="serialwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>376</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>265</width>
    <height>376</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:15px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,2">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QFrame" name="formFrame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>221</width>
       <height>261</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="rowWrapPolicy">
       <enum>QFormLayout::DontWrapRows</enum>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="horizontalSpacing">
       <number>9</number>
      </property>
      <property name="verticalSpacing">
       <number>9</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#006c94</string>
        </property>
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
        <item>
         <widget class="QPushButton" name="pushButton_refresh">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/new/prefix1/refresh.png);
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_number">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox{
	
	color: #78c2fa;
    background-color: #fffff;
    border-radius: 5px;
    border: 2px solid #78c2fa;
    padding-left: 2px;
}
 
QComboBox:disabled{
    background:#006c94;
    border: 1px solid #006c94;
}
   </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_baud">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
	
	color: #78c2fa;
    background-color: #fffff;
    border-radius: 5px;
    border: 2px solid #78c2fa;
    padding-left: 2px;
}
 
QComboBox:disabled{
    background:#006c94;
    border: 1px solid #006c94;
}
   


</string>
        </property>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_data">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
	
	color: #78c2fa;
    background-color: #fffff;
    border-radius: 5px;
    border: 2px solid #78c2fa;
    padding-left: 2px;
}
 
QComboBox:disabled{
    background:#006c94;
    border: 1px solid #006c94;
}
   </string>
        </property>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="comboBox_check">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
	
	color: #78c2fa;
    background-color: #fffff;
    border-radius: 5px;
    border: 2px solid #78c2fa;
    padding-left: 2px;
}
 
QComboBox:disabled{
    background:#006c94;
    border: 1px solid #006c94;
}
   </string>
        </property>
        <item>
         <property name="text">
          <string>无</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>奇校验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>偶校验</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboBox_stop">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
	
	color: #78c2fa;
    background-color: #fffff;
    border-radius: 5px;
    border: 2px solid #78c2fa;
    padding-left: 2px;
}
 
QComboBox:disabled{
    background:#006c94;
    border: 1px solid #006c94;
}
   </string>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color : #006c94;</string>
        </property>
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#006c94</string>
        </property>
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#006c94</string>
        </property>
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#006c94</string>
        </property>
        <property name="text">
         <string>串口号</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_serialport_open">
     <property name="minimumSize">
      <size>
       <width>221</width>
       <height>51</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton
{
    color: #78c2fa;
    background-color: #FFFFFF;
    border-color: #78c2fa;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}
QPushButton::hover
{	
    color: #FFFFFF;
    background-color: #78c2fa;
    border-color: #78c2fa;
}
QPushButton::pressed,QPushButton::checked
{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #78c2fa, stop:1 #006c94);
}</string>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
